docker/start.py

```diff
@@ -13,7 +13,7 @@
 
 # Utility functions
 def log(txt: str) -> None:
-    print(txt, file=sys.stderr)
+    print(txt, flush=True)
```

Comment:
`start.py` is also used by the non-worker Dockerfile. Is this change intentional here, given that you are only targetting Dockerfile-workers?

Note that this change also makes it so that the `error` function below writes to stdout, not stderr---which sounds wrong to me.

Reply:
[Docker Documentation](https://docs.docker.com/config/containers/logging/).
>`STDOUT` is usually a command’s normal output, and `STDERR` is typically used to output error messages. By default, docker logs shows the command’s `STDOUT` and `STDERR`.

I was only trying to make sure it all worked the same way. The `error()` subroutine prints and exits, in both files. If you like, put it back the way it was. I did offer that above in the original comment. But it did feel wrong to me for `log()` to print to stderr by default.
>If someone would rather I switch to using stderr, leave me a comment to that effect and I'll adjust.

Perhaps a compromise of:  (from start.py)

```sh
# Utility functions
def log(txt: str) -> None:
    print(txt, flush=True)


def error(txt: str) -> NoReturn:
    print(txt, file=sys.stderr)
    sys.exit(2)
```

Duplicated across both? Maintaining the comments in configure....py(are those what are called docstrings?) of course.

Reply:
Nevermind, discussion continued below.